code/modules/vore/eating/bellymodes_vr.dm

```diff
@@ -121,12 +121,22 @@
 			for(var/slot in slots)
 				var/obj/item/thingy = M.get_equipped_item(slot = slot)
 				if(thingy)
-					M.unEquip(thingy,force = TRUE)
-					thingy.forceMove(owner)
+					if(!M.canUnEquip(thingy))
+						continue
+					if(thingy == M.get_equipped_item(slot_w_uniform))
```

Comment:
Why do this? Seems more expensive than just looking at the slot you selected earlier, since if this is the case, slot would also equal slot_w_uniform (though this would be handled for you if you just fix unEquip)